lumber-generate.js

```diff
@@ -10,10 +10,15 @@ const logger = require('./services/logger');
 program
   .description('Generate a backend application with an ORM/ODM configured.')
   .option('-c, --connection-url <connectionUrl>', 'Enter the database credentials with a connection URL')
+  .option('--silent', 'Do not print any logs')
```

Comment:
A new option?
Why don't we keep things simple and remove those logs.

+ it won't be silent as the "Hooray" log remains printed.

Reply:
We want to simplify Lumber in the near future, but such implementation is going to the opposite direction :/